nexus-client/src/lib.rs

```diff
@@ -128,7 +128,7 @@ impl From<omicron_common::api::external::InstanceCpuCount>
     for types::InstanceCpuCount
 {
     fn from(s: omicron_common::api::external::InstanceCpuCount) -> Self {
-        Self(s.0)
+        Self(s.0.into())
```

Comment:
It's not immediately clear to me why this is required. It seems like custom defining the schema is causing the type to be generated as `i64` instead of the previous `u16int`. The _schema_ looks correct, so I'm uncertain if this is a progenitor issue. I'll try debugging the generated types.

Reply:
I don't *think* this still exists in the latest commit, right? Not seeing anything changed in the `clients/nexus` module

Reply:
Yeah, I figured out how to avoid it.